Walletbeat 2025 roadmap

polymutex opened the pull request.

darrylyeo reviewed (commented):
Link usernames to GitHub profiles?

polymutex committed:
Flesh out roadmap some more.

polymutex committed:
Initial roadmap for Walletbeat in 2025.

polymutex committed:
Fill in a few more things on governance.

polymutex committed:
Add EIP-7702 tracker, flesh out governance structure more.

polymutex committed:
Start to write EF project grant proposal for Walletbeat 2025 launch.

nconsigny committed:
Update proposal.md

Assign hww review to PatrickAlphaC (as he is already reviewing hardware extensively) and add review for Zerion & Phantom (hugoo)

nconsigny committed:
Update proposal.md

polymutex committed:
Update grant application in light of discussions with contributors.

polymutex committed:
Update roadmap and synchronize with EF grant application priorities.

polymutex committed:
Fix date.

polymutex committed:
Add a few clarifications.
